### DISCUSSION TITLE
Override seminars templates

### DISCUSSION THREAD

**Discussion by wmarie-joseph** [oliverklee/ext-seminars#2406] [General]
Hello,

I'm using TYPO3 11.5.24 with PHP 8.1 and seminars version 5.1.

i'm trying to override the templates, i managed to override the list view using this typoscript :

plugin.tx_seminars_pi1 {
  templateFile = {template_file_path}
}

I want to override other templates, like the detailed view, is this possible ?

**Comment by oliverklee** [comment]
Yes, this is possible. The template file configured in your code snipped (i.e., `EXT:seminars/Resources/Private/Templates/FrontEnd/FrontEnd.html` by default) also includes the HTML for the single view (and all other non-Fluid views).

The Fluid views are configured according to the Extbase/Fluid conventions.

**Reply by wmarie-joseph** [reply]
@oliverklee hi,

I would like to override multiple template files, is this possible with using this setup ?

plugin.tx_seminars_pi1 {
templateFile = {template_file_path}
templateFile2 = {template_file_path2}
}

For example i would also like to override this file :

EXT:seminars/Resources/Private/Templates/EventRegistration/Confirm.html

**Reply by oliverklee** [reply]
First, you'll need to know which templates you'd like to override - the old marker-based templates or the Fluid templates.

https://stackoverflow.com/questions/50512282/override-a-fluid-template-from-another-extension describes how to override the Fluid templates. (You then provide the paths of the folders, not the path to the individual templates.)

To override the marker-based templates, use the  `*.templateFile` settings as described on https://docs.typo3.org/p/oliverklee/seminars/5.1/en-us/Reference/SetupCommonForFront-endPlug-inAndBack-endModuleInPlugintxSeminars/Index.html and https://docs.typo3.org/p/oliverklee/seminars/5.1/en-us/Reference/SetupForTheSeminarManagerFront-endPlug-inInPlugintxSeminarsPi1/Index.html.

**Reply by wmarie-joseph** [reply]
@oliverklee 

Thank you !

**Comment by wmarie-joseph** [comment]
Hello again,

Since the last time, i managed to extend the "seminars" extension to add additional custom fields. (using a custom extension created with extension builder).

My issue now is, how can i use and display my additional data in the "seminars" templates without having to modify the default code of seminars ?

**Reply by oliverklee** [reply]
In addition to using a custom HTML template, you'll need to use one of the hooks (for the list view, single view etc.) to set your additional  markers or subparts in the template: https://docs.typo3.org/p/oliverklee/seminars/5.1/en-us/Reference/Hooks/Index.html
